src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinatorModel.swift

```diff
@@ -31,8 +31,10 @@ class ExposureSubmissionCoordinatorModel {
 
 	let exposureSubmissionService: ExposureSubmissionService
 	let coronaTestService: CoronaTestService
-
+	let eventProvider: EventProviding
+	
 	var coronaTestType: CoronaTestType?
+	var markAsUnseen: Bool = false
```

Comment:
Hmm I don't unterstand what is marked as unseen here. 
Can you explain it to me or have a more self-explaining name?

Reply:
Renamed it